.vsts-ci/windows/templates/windows-packaging.yml

```
@@ -4,7 +4,7 @@ parameters:
   - name: jobName
     default: 'win_packaging'
   - name: runtimePrefix
-    default: 'win7'
+    default: 'win81'
```

```suggestion
    default: 'win8'
```

To support Windows Server 2012.

---

See the PR https://github.com/PowerShell/PowerShell/pull/18322. We reverted win7 -> win81 change.  win7 will still not be supported.

---

But will we support Windows Server 2012? see: https://github.com/PowerShell/PowerShell/pull/18354#issuecomment-1287899773.

---

@adityapatwardhan I think what @xtqqczze is saying is that to support Server 2012 (which is still under Extended support), we need `win8`, not `win81`